package.json

```diff
@@ -14,10 +14,12 @@
     "start": "tsdx watch",
```

Comment:
I'm always triggering myself. Can we rename this to `dev`. I know it's 100% unrelated 😂

Reply:
Honestly same, changed